### PR TITLE
Fixed being unable to superfart while in soft crit

### PR DESCRIPTION
**Pull request by JohnGinnane**
:cl: JohnGinnane
fix: Fixed being unable to superfart while in soft crit
/:cl:

When I added the check for `MODE_WHISPER` in #7517 I inadvertently disabled using emotes via say, e.g. "say *superfart". This lets us use emotes while in softcrit now

### CODE REVIEW

File: hippiestation.dme [hunk 1]
```diff
@@ -2737,6 +2736,7 @@
 #include "hippiestation\code\game\objects\items\implants\implant_teleporter.dm"
 #include "hippiestation\code\game\objects\items\implants\implantcase.dm"
 #include "hippiestation\code\game\objects\items\implants\implanter.dm"
+#include "hippiestation\code\game\objects\items\melee\energy.dm"
```
Comment (JohnGinnane) [inline hippiestation.dme:2739]: We can ignore this, it's probably that someone else's hippiestation.dme wasn't in order when we merged the PR and my dreammaker has automatically reorganised things

File: hippiestation/code/modules/mob/living/say.dm [hunk 2]
```diff
@@ -15,7 +15,10 @@
 
 /mob/living/say(message, bubble_type,var/list/spans = list(), sanitize = TRUE, datum/language/language = null, ignore_spam = FALSE)
 	// If we're in soft crit and tried to talk, automatically make us whisper
-	if (stat == SOFT_CRIT && get_message_mode(message) != MODE_WHISPER)
-		message = "#" + message
+	if (length(message) > 2)
+		var/first_char = copytext(message, 1, 2)
+
+		if (first_char != "*" && stat == SOFT_CRIT && get_message_mode(message) != MODE_WHISPER)
```
Comment (JohnGinnane) [inline hippiestation/code/modules/mob/living/say.dm:21]: I think this is a *little* snowflakey but I don't see if there's define for emotes, other than `check_emote()`

Comment (ghost) [reply hippiestation/code/modules/mob/living/say.dm:21]:
```
/mob/proc/emote(act, m_type = null, message = null)
	act = lowertext(act)
	var/param = message
	var/custom_param = findchar(act, " ")
	if(custom_param)
		param = copytext(act, custom_param + 1, length(act) + 1)
		act = copytext(act, 1, custom_param)

	var/datum/emote/E
	E = E.emote_list[act]
	if(!E)
		to_chat(src, "<span class='notice'>Unusable emote '[act]'. Say *help for a list.</span>")
		return
	E.run_emote(src, param, m_type)
```
https://github.com/HippieStation/HippieStation/blob/8cf20740757235f8d798649f7bedd33ea1c65acd/code/modules/mob/emote.dm#L2
Doesn't seem to be a text proc for checking if a message is an emote

Comment (JohnGinnane) [reply hippiestation/code/modules/mob/living/say.dm:21]: nah it's here:

https://github.com/HippieStation/HippieStation/blob/1bc95b04765de098ab0196cdfea95c8aa9112367/code/modules/mob/living/say.dm#L125
https://github.com/HippieStation/HippieStation/blob/05006554876d1486cbe14b886183850f7de33e99/code/modules/mob/say.dm#L76

**Review by Carbonhell — APPROVED** [review]

**Pull request merged**